src/checkov/checkovRunner.ts

```diff
@@ -23,7 +24,14 @@ const getDockerFileMountParams = (mountDir: string, filePath: string | undefined
     return ['-v', `"${absPath}:${mountDir}/${baseName}"`];
 };
 
-const getDockerRunParams = (workspaceRoot: string | undefined, filePath: string, extensionVersion: string, configFilePath: string | undefined, checkovVersion: string | undefined, prismaUrl: string | undefined, certPath: string | undefined, debugLogs: boolean | undefined) => {
+const getPathParamsForDockerRun = (mountDir: string, filePath: string | undefined, flag: string): string[][] => {
```

Comment:
I think the type should be
```suggestion
const getPathParamsForDockerRun = (mountDir: string, filePath?: string, flag: string): string[] => {
```

Reply:
it return array of arrays